WooCommerce/Classes/ViewRelated/Products/Edit Product/Edit Price/ProductPriceSettingsViewController.swift

```diff
@@ -478,6 +505,17 @@ private extension ProductPriceSettingsViewController {
         Section(title: NSLocalizedString("Tax Settings", comment: "Section header title for product tax settings"), rows: [.taxStatus, .taxClass])
         ]
     }
+
+    func getDecimalPrice(_ price: String?) -> NSDecimalNumber? {
+        guard let price = price else {
+            return nil
+        }
+        let currencyFormatter = CurrencyFormatter()
+        let currencyCode = CurrencySettings.shared.currencyCode
+        let unit = CurrencySettings.shared.symbol(from: currencyCode)
+        let value = price.replacingOccurrences(of: unit, with: "")
```

Comment:
nit: the above lines seem very useful for future reuse, do you think we could either add these to the existing `currencyFormatter.convertToDecimal(from:)` with a currency unit parameter, or a separate `CurrencyFormatter` helper?

Reply:
I moved the logic inside the method `convertToDecimal`. In this way, we can use this same method in other parts of the app, where we do the currency symbol replacing. 👍